sensors/start_button.py

```diff
@@ -0,0 +1,38 @@
+import RPi.GPIO as GPIO
+
+class StartButton:
+    """
```

Comment:
is this another way of adding comments? Like # or 
*/ /*

Reply:
Yup and its the "standard" way to add docstrings: https://peps.python.org/pep-0257/

Technically it's a string literal that's ignored because it's not assigned to any variable...

Reply:
oh okay good to know, well done then I didnt know